pkg/spdk/replica.go

```diff
@@ -193,7 +199,8 @@ func (r *Replica) replicaLvolFilter(bdev *spdktypes.BdevInfo) bool {
 		return false
 	}
 	lvolName := spdktypes.GetLvolNameFromAlias(bdev.Aliases[0])
-	return IsReplicaLvol(r.Name, lvolName) || (len(r.ActiveChain) > 0 && r.ActiveChain[0] != nil && r.ActiveChain[0].Name == lvolName)
+	// it is okay to habe backing image snapshot in the results, because we exclude it when finding root or construct the snapshot map
```

Comment:
```suggestion
	// it is okay to have backing image snapshot in the results, because we exclude it when finding root or construct the snapshot map
```